samd/samd21/timers.c

```diff
@@ -61,6 +61,7 @@ void turn_on_clocks(bool is_tc, uint8_t index, uint32_t gclk_index) {
     }
     PM->APBCMASK.reg |= 1 << clock_slot;
     _gclk_enable_channel(gclk_id, gclk_index);
+    set_timer_handler(index, timer_handler);
```

Comment:
No need so pass this through. Let the CircuitPython side worry about the handler.

Reply:
Makes sense. I think I initially did this to just save an extra line of code, but I can't remember.